slides/intro-02-data-budget.qmd

```diff
@@ -276,89 +263,103 @@ Make a plot or summary and then share with neighbor
 
 ## 
 
-```{r taxi-tip-counts}
+```{r forested-forested-counts}
 #| fig-align: 'center'
-taxi_train %>% 
-  ggplot(aes(x = tip)) +
+forested_train %>% 
+  ggplot(aes(x = forested)) +
   geom_bar()
 ```
 
 ## 
 
-```{r taxi-tip-by-local}
+```{r forested-by-tree-no-tree}
 #| fig-align: 'center'
-taxi_train %>% 
-  ggplot(aes(x = tip, fill = local)) +
+forested_train %>% 
+  ggplot(aes(x = forested, fill = tree_no_tree)) +
   geom_bar() +
   scale_fill_viridis_d(end = .5)
 ```
 
 ## 
 
-```{r taxi-tip-by-hour}
+```{r forested-by-precip-annual}
 #| fig-align: 'center'
-taxi_train %>% 
-  ggplot(aes(x = hour, fill = tip)) +
-  geom_bar()
+#| message: false
+#| warning: false
+forested_train %>% 
+  ggplot(aes(x = precip_annual, fill = forested, group = forested)) +
+  geom_histogram() +
+  scale_fill_viridis_d(end = .5)
 ```
 
+<!-- TODO: should we `position = "identity", alpha = .5` here? -->
```

Comment:
New TODO: should we set `position = "identity"` when plotting `forested` vs `precip_annual`?